backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/scheduler/Scheduler.java

```diff
@@ -33,8 +35,11 @@ public Scheduler(PublishNormUseCase publishNormUseCase) {
    * The cron schedule is controlled by the property {@code publish.cron}.</p>
    */
   @Scheduled(cron = "${publish.cron}")
+  @SchedulerLock(name = "scheduledPublishToBucket", lockAtMostFor = "240m", lockAtLeastFor = "120m")
```

Comment:
Didn't read through the documentation but shouldn't we be setting the lockAtMostFor to a conservative value just above the longest tie we expect the task to run? Since we don't know yet how long will the job be running not sure yet if we should even say a couple of hours... The lockAtLeastFor can be a few minutes I think.
Not sure about the releasing of the lock if the other pod will try even later to run the publication job?

Reply:
Here's a link to the documentation: https://github.com/lukas-krecan/ShedLock 

lockAtLeastFor:
It states: "Lastly, you can set lockAtLeastFor attribute which specifies minimum amount of time for which the lock should be kept. Its main purpose is to prevent execution from multiple nodes in case of really short tasks and clock difference between the nodes."
I think the value can be as high as 24 hours  without any damage since we only run the job once a day. Since our pods are run on one cluster we shouldn't have clock issues and the value could also be as low as one minute.
To be on the save side I decided for 2h but I think it doesn't really matter.

lockAtMostFor:
I only guessed here how long it could take at most plus some buffer time. Maybe we can adjust the value once we know how long it will run?